### ISSUE TITLE
Status Bar Causes Undesired Resizing of Split Windows

### ISSUE THREAD

**Issue by adrian-enspired** [Komodo/KomodoEdit#2651]
### Short Summary
The split window width changes depending on the content displayed in the status bar.

### Steps to Reproduce
- Open some file with a substantial amount of text and split the view.  Rotated the split so it is vertical (side-by-side windows).  Leave the split at its default position, or adjust as desired.
- Begin selecting text in one window.  At times, the effect can also be seen by simply moving the cursor position (but this is less obvious).

### Expected results
The size of each split window should not change.

### Actual results
The size of the split windows change as the size of the content in the status bar changes.  Note, this is not simply related to overflow; the issue still occurs when there is plenty of free space in the status bar.

demonstration:  https://youtu.be/u1_Tq8yuzyo

### Platform Information

*Komodo Edit or IDE?* 
Komodo Edit

*Komodo Version?* 
10.2

*Operating System (and version)?* 
Fedora 25 (gnome 3)

### Additional Information
Please see https://community.komodoide.com/t/komodo-10s-new-status-bar-a-couple-of-items-for-discussion/2677/12?u=adrian_enspired and the following conversation.

**Comment by Naatan** [comment]
Thanks for filing this @adrian-enspired !

**Comment by cgchoffman** [comment]
@Naatan, were you able to reproduce this?  I'm not seeing it in Edit or IDE.

**Comment by Naatan** [comment]
I don't believe so, not sure. There's more info on the forum thread referenced: https://community.komodoide.com/t/komodo-10s-new-status-bar-a-couple-of-items-for-discussion/2677/28

**Comment by cgchoffman** [comment]
I'm not seeing any helpful/additional information.  I tried to make the window as narrow as possible and opened one of the largest files I can find in the Komodo repo.  I selected from the start to a few thousand lines down and the splitview width never changes.

@adrian-enspired, are you still seeing this?

**Comment by adrian-enspired** [comment]
Sorry for the late reply.  

Yes, I just installed a fresh copy of komodo edit (11.1.0, build 18196) and the behavior persists, exactly as shown in the screen recording above.

**Comment by adrian-enspired** [comment]
BTW updated workaround:
```css
[anonid=statusbar-selection] {
  display: none;
}
```

this is the minimal amount of hiding I've been able to find that gets rid of the issue. might help diagnose.

**Comment by adrian-enspired** [comment]
@cgchoffman Just wanted to tag you to be sure you saw my reply, as this issue is still backlogged "pending response."  I switched themes today and forgot to add my workaround above… problem definitely still exists  : )

**Comment by cgchoffman** [comment]
> that was a completely stock installation and was only a few days old at the time I made that screencast.

It's not a stock installation in the video.  The default UI doesn't look like that.  Your status bar is at the bottom (Prefs > Appearance > Move Status bar to bottom), you have the additional status bar info (don't know how those get there but if you have a userscript that runs at start that would explain why you don't think you added them), among other things.  Can you reset your profile and see if you can reproduce the issue?  *Help > Troubleshooting > Restart in Safe-Mode*

**Comment by adrian-enspired** [comment]
Sorry; "stock" meaning I had not installed any user scripts/plugins/etc.. Everything I had was provided from the official install script. Yes, I had moved the status bar to the bottom; no, I did not do anything to add extra info on the statusbar.

This issue has come up consistently over several installs on both linux (ubuntu) and osx over the past 18 months. I've had to add the custom css to work around it, every time.

> Can you reset your profile and see if you can reproduce the issue?

Yes, will do tomorrow. I'll also find some time to do a clean install and make a new screencast.

Thanks for your time.

**Comment by cgchoffman** [comment]
@adrian-enspired, ya the key I'm missing is how you get Komodo to a state where I can repro.  I don't want to add your patch for something if it's not part of the core.  I appreciate you checking a clean profile when you have a chance, thanks for that.

**Comment by adrian-enspired** [comment]
https://youtu.be/rxwGe2H8KSc

**Comment by adrian-enspired** [comment]
Also, the css I pasted above is definitely not a "patch"  :smile:  
It just hides a component so the resizing stops. Workaround, but doesn't fix anything.

**Comment by cgchoffman** [comment]
Thanks for the video.  Maybe I have something removing those status bar field.  Not sure.
